internal/command/plan.go

```diff
@@ -247,33 +247,42 @@ Plan Customization Options:
 
 Other Options:
 
-  -compact-warnings   If Terraform produces any warnings that are not
-                      accompanied by errors, shows them in a more compact form
-                      that includes only the summary messages.
+  -compact-warnings          If Terraform produces any warnings that are not
+                             accompanied by errors, shows them in a more compact
+                             form that includes only the summary messages.
 
-  -detailed-exitcode  Return detailed exit codes when the command exits. This
-                      will change the meaning of exit codes to:
-                      0 - Succeeded, diff is empty (no changes)
-                      1 - Errored
-                      2 - Succeeded, there is a diff
+  -detailed-exitcode         Return detailed exit codes when the command exits.
+                             This will change the meaning of exit codes to:
+                             0 - Succeeded, diff is empty (no changes)
+                             1 - Errored
+                             2 - Succeeded, there is a diff
 
-  -input=true         Ask for input for variables if not directly set.
+  -generate-config-out=path  (Experimental) If import blocks are present in
+                             config, instructs Terraform to generate HCL for any
```

Comment:
I noticed in this file we spell out "configuration" so adding a suggestion:
```suggestion
                             the current configuration, this command instructs Terraform to generate HCL for any
```